src/order/filter/basic.lean

```diff
@@ -969,6 +969,10 @@ lemma eventually_of_forall {p : α → Prop} {f : filter α} (hp : ∀ x, p x) :
   ∀ᶠ x in f, p x :=
 univ_mem' hp
 
+lemma forall_eventually_of_eventually_forall {f : filter α} {p : α → β → Prop}
```

Comment:
This is one direction of `eventually_all` from below, right?

Reply:
Yes, this is the direction of `eventually_all` that remains true without the `[finite ι]` assumption.

(I'm stunned that this lemma does not already exist btw.)